doc/devices/runtime.rst

```diff
@@ -0,0 +1,56 @@
+Qiskit Runtime Programs
+=======================
+
+PennyLane-Qiskit supports running PennyLane on IBM Q hardware via the Qiskit runtime programs ``circuit-runner``
+and ``sampler``. You can choose between those two runtime programs and also have the possibility to choose the
+backend on which the circuits will be run. Those two devices inherit directly from the ``IBMQ`` device and work the
+the same way, you can refer to the corresponding documentation for details about token and providers.
```

Comment:
Adding a link to the referenced documentation could be useful here.

Reply:
How can you refer to another page generated in the documentation?

Reply:
One possibility is to link to the live page:`<https://pennylaneqiskit.readthedocs.io/en/latest/devices/ibmq.html>`__`. Not sure about the syntax for referencing a newly built page with Sphinx, but I'd think it's possible.

Reply:
@antalszava Thank you I'll do it like that!